From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Wed, 13 Aug 2025 06:31:47 +0000
Subject: [PATCH] [HACK] core: pm: use ro.boot.hwname for sku specific app
 disabling

ro.boot.hardware.sku does not work on blossom

Change-Id: I2b2af60a1d340bc58e2918cad56d861fe72f79c8
Signed-off-by: fukiame <fukiame@proton.me>
---
 .../core/java/com/android/server/pm/PackageManagerService.java  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/java/com/android/server/pm/PackageManagerService.java b/services/core/java/com/android/server/pm/PackageManagerService.java
index 7a97e474aa61..7304b5bb9e23 100644
--- a/services/core/java/com/android/server/pm/PackageManagerService.java
+++ b/services/core/java/com/android/server/pm/PackageManagerService.java
@@ -4404,7 +4404,7 @@ public class PackageManagerService implements PackageSender, TestUtilityService
         if (ArrayUtils.isEmpty(apkList)) {
            return;
         }
-        String sku = SystemProperties.get("ro.boot.hardware.sku");
+        String sku = SystemProperties.get("ro.boot.hwname");
         if (!TextUtils.isEmpty(sku) && ArrayUtils.contains(skuArray, sku)) {
             return;
         }
